dvc/command/daemon.py

```diff
@@ -24,10 +25,10 @@ def run(self):
 
 class CmdDaemonAnalytics(CmdDaemonBase):
     def run(self):
-        from dvc.analytics import Analytics
+        from dvc import analytics
 
-        analytics = Analytics.load(self.args.target)
-        analytics.send()
+        analytics.send(self.args.target)
+        os.remove(self.args.target)
```

Comment:
The convention about creating and deleting a report file seems to belong to analytics.py, maybe we should delete the file there too?

Reply:
that's more like the communication method used when demonizing stuff. Store things in a file, then remove them.

Reply:
Also, it is not expected that a `send` method has a side effect of removing the file you are sending :sweat_smile:

Reply:
@mroutis you can call the menthod `send_and_delete` if you want to.

Reply:
Still don't like the idea, @efiop , but I documented it better on `analytics.collect_and_send_report` :)